sorting/Bogo_Sort.cpp

```diff
@@ -41,7 +53,8 @@ std::vector <int> shuffle(std::vector <int> arr)
     return arr;
 }
 //Randomized bogo sort
```

Comment:
```suggestion

/**
 * Implement randomized Bogosort algorithm and sort the elements of a given array.
 * @tparam T typename of the array
 * @tparam N length of array
 * @param arr array to sort
 * @returns new array with elements sorted from a given array
 */
```
Same as above, this function is a pass-by-value